### PR TITLE
Add ESLint and JSCS

### LINKED ISSUE #2: Add jscs
Use jscs-preset-gulp

### LINKED ISSUE #3: Explicit eslint usage
To bring inline with gulp repos. Use eslint-config-gulp and update code style

### LINKED ISSUE #4: Update repo location references
In package.json, readme, etc

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #2 
Fixes #3 
Fixes #4

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-  },
-  "rules": {
-    "semi": [2, "always"],
-    "quotes": [2, "single"],
-    "strict": [2, "global"],
-    "no-underscore-dangle": 0,
-    "consistent-return": 0
-  },
+  "extends": "gulp",
   "env": {
```
Comment (phated) [inline .eslintrc]: these can be removed

File: .eslintrc [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
     "es6": true,
-    "browser": true,
-    "node": true
+    "browser": true
+  },
+  "ecmaFeatures": {
```
Comment (phated) [inline .eslintrc]: this can be removed

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
-sudo: false
 language: node_js
+
+sudo: false
```
Comment (phated) [inline .travis.yml:1]: prefer sudo: false to be at the top

File: package.json [hunk 4]
```diff
@@ -15,6 +15,7 @@
     "index.js"
   ],
   "scripts": {
+    "lint": "eslint . && jscs *.js test/",
     "test": "lab -cvL"
```
Comment (phated) [inline package.json:20]: remove linting from the lab command and put explicit call in pretest

**Comment by pdehaan** [comment]
Done.

**Pull request merged**